### PR TITLE
News endring i tjeneste konfigurasjon

### PR DESCRIPTION
**Pull request by akcssb**
Oppdatert tjeneste-artiklene med nye bilder og forklaringer da tjenstekonfigurasjonen har endret seg.

Fjernet en del tekst og henviste til jupyter.qmd der det var hensiktsmessig.

Skriver post på Dapla Nyheter så fort disse endringene er merget.

- - -
This change is [Reviewable](https://reviewable.io/reviews/statisticsnorway/dapla-manual/175)

### CODE REVIEW

**Comment by akcssb** [comment]
Legger til nye skjermbilder

File: dapla-manual/images/dapla-lab-jdemetra-konf-service.png
Comment (skykanin) [inline dapla-manual/images/dapla-lab-jdemetra-konf-service.png:1]: This image is outdated, this is what it looks like on prod now:

image

File: dapla-manual/statistikkere/jupyter-pyspark.qmd [hunk 1]
```diff
@@ -21,60 +21,7 @@ Før man starter **Jupyter-pyspark** bør man ha lest [kapitlet om Dapla Lab](./
...
-
-@fig-dapla-lab-konf-git viser at brukeren som standard får aktivert **Aktiver Git**. Dette innebærer at Git-brukernavn, Git e-post og GitHub-token arves fra brukerkonfigurasjonen. I tillegg opprettes SSBs standard Git-konfigurasjon i `~/.gitconfig`. 
-
-## Konfigurering av PySpark-ressurser i Notebook
-
-Fra og med den siste oppdateringen trenger du ikke lenger å manuelt angi mengden RAM i PySpark-konfigurasjonen. Tjenesten er nå konfigurert slik at Spark automatisk tildeles 90% av det totale minnet som er definert i tjenestekonfigurasjonen under **Ressurser**.
-
-Dette forenkler oppsettet og sikrer optimal utnyttelse av tilgjengelige ressurser.
-
-
```
Comment (skykanin) [inline dapla-manual/statistikkere/jupyter-pyspark.qmd]: I think the jupyter-pyspark specific configuration should still be included. CC: @ssb-jnk

**Review by skykanin — CHANGES_REQUESTED** [review]
Some small changes to be made, but otherwise this looks good 👍

File: dapla-manual/statistikkere/datadoc.qmd [hunk 2]
```diff
@@ -5,7 +5,9 @@ date-modified: "11/26/2024"
 
 ![](../images/dapla-lab-datadoc-service.png){style="max-width: 46%; float: right;" fig-alt="Jupyter Service"}
 
-Datadoc editor er et tjeneste som tilbyr et grafisk grensesnitt for å dokumentere datasett og variablene som utgjør datasettet. Formålet med tjenesten er å tilby et lett-å-bruke grensesnitt som hovedsakelig vil benyttes første gang man dokumenterer en type datasett.
+Datadoc er et grafisk grensesnitt for å dokumentere datasett og variablene som utgjør datasettet.
```
Comment (Jorgen-5) [inline dapla-manual/statistikkere/datadoc.qmd]:
```suggestion
Datadoc editor er et grafisk grensesnitt for å dokumentere datasett og variablene som utgjør datasettet.
```

File: dapla-manual/statistikkere/datadoc.qmd [hunk 3]
```diff
@@ -23,22 +25,25 @@ Datadoc editor bruker ca. 1 minutt på starte og etter det klart for dokumentere
 
 ## Konfigurasjon
 
-Før man starter Datadoc editor bør man konfigurere tjenesten. Dette er spesielt viktig siden du bare kan representere et Dapla-team for hver Datadoc editor man starter. I tjenestekonfigurasjonen til Datadoc editor er det 2 faner: **Dapla** og **Tjeneste**.
+Før man starter Datadoc bør man konfigurere tjenesten. Dette er spesielt viktig siden du bare kan representere et Dapla-team for hver Datadoc editor man starter. I tjenestekonfigurasjonen til Datadoc er det to nedtrekksmenyer: **Data** og **Tjeneste**.
```
Comment (Jorgen-5) [inline dapla-manual/statistikkere/datadoc.qmd]:
```suggestion
Før man starter Datadoc editor bør man konfigurere tjenesten. Dette er spesielt viktig siden du bare kan representere et Dapla-team for hver Datadoc editor man starter. I tjenestekonfigurasjonen til Datadoc er det to nedtrekksmenyer: **Data** og **Tjeneste**.
```

**Review by Jorgen-5 — DISMISSED** [review]
We changed the datadoc name to datadoc editor. I found a few places where editor was missing. Other than that the datadoc chapter was very nice!

File: dapla-manual/statistikkere/datadoc.qmd [hunk 4]
```diff
@@ -23,22 +25,25 @@ Datadoc editor bruker ca. 1 minutt på starte og etter det klart for dokumentere
...
 
-### Dapla
+@fig-datadoc-config viser tilfellet der det er valgt å representere tilgangsgruppen **developers** i teamet **Dapla Felles**, derav *dapla-felles-developers*. Dette er standardvalget. 
 
-Under fanen **Dapla** i **Datadoc editor konfigurasjoner** er det en nedtrekksmeny der man kan velge **Team og tilgangsgruppe**. I nedtrekksmenyen får du listet alle team og tilgangsgrupper du er med i. Listen i nedtrekksmenyen vises på formen  **\<daplateam\>-\<tilgangsgruppe\>**. @fig-datadoc-config viser tilfellet der det er valgt å representere tilgangsgruppen **developers** i teamet **Dapla Felles**, derav **dapla-felles-developers**. Det er også det som velges som standard hvis man ikke velger noe annet. 
+![Data-menyen i tjenestekonfigurasjonen for Datadoc editor.](../images/datadoc-config.png){fig-alt="Slik ser config for datadoc ut" width="100%" #fig-datadoc-config}
 
-![Dapla-fanen i tjenestekonfigurasjonen for Datadoc editor.](../images/datadoc-config.png){fig-alt="Slik ser config for datadoc editor ut" width="100%" #fig-datadoc-config}
+Datadoc editor for øyeblikket ikke kildedata selv om man kan velge begrunnelse og tilgangvarighet fra konfigurasjons fanen.
```
Comment (Jorgen-5) [inline dapla-manual/statistikkere/datadoc.qmd]:
```suggestion
Datadoc editor støtter for øyeblikket ikke kildedata selv om man kan velge begrunnelse og tilgangvarighet fra konfigurasjons fanen.
```

**Comment by akcssb** [comment]
All done:=) @skykanin @Jorgen-5

File: dapla-manual/images/dapla-lab-jdemetra-konf-service.png
Comment (skykanin) [inline dapla-manual/images/dapla-lab-jdemetra-konf-service.png:1]: This image is still outdated

Comment (skykanin) [inline dapla-manual/images/dapla-lab-jdemetra-konf-service.png:1]: This image is still outdated

Comment (skykanin) [reply dapla-manual/images/dapla-lab-jdemetra-konf-service.png:1]: `dapla-manual/images/dapla-lab-jdemetra-konf-service.png`

**Review by skykanin — CHANGES_REQUESTED** [review]

File: dapla-manual/images/dapla-lab-jdemetra-konf-service.png
Comment (akcssb) [reply dapla-manual/images/dapla-lab-jdemetra-konf-service.png:1]: my bad. forgot to delete the image. it is now deleted.

**Review by skykanin — APPROVED** [review]
Approved

**Pull request merged**